cmd/webhook/main.go

```diff
@@ -320,12 +321,12 @@ func NewConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 					sourcesv1beta1_:  &sourcesv1beta1.ApiServerSource{},
 				},
 			},
-			sourcesv1alpha2.Kind("PingSource"): {
+			sourcesv1beta1.Kind("PingSource"): {
 				DefinitionName: sources.PingSourceResource.String(),
-				HubVersion:     sourcesv1alpha1_,
+				HubVersion:     sourcesv1alpha2_,
```

Comment:
shouldn't the hub be v1alpha1?

Reply:
next PR will remove v1alpha1 since it's not stored, nor served.

Reply:
ack, didn't know it wasn't stored nor served... SGTM